dmscripts/generate_framework_agreement_signature_pages.py

```diff
@@ -95,28 +108,65 @@ def render_html_for_suppliers_awaiting_countersignature(rows, framework, templat
         data['countersigned_at'] = datetime.strptime(
             data['countersigned_at'], '%Y-%m-%dT%H:%M:%S.%fZ'
         ).strftime('%d %B %Y')
+        if data['signed_agreement_returned_at']:
+            data['signed_agreement_returned_at'] = datetime.strptime(
+                data['signed_agreement_returned_at'], '%Y-%m-%dT%H:%M:%S.%fZ'
+            ).strftime('%d %B %Y')
```

Comment:
Why do we need to update this if it's already set?

Reply:
This is to reformat the date for display converting it from the format returned by the API.
This follows the same pattern as https://github.com/alphagov/digitalmarketplace-scripts/blob/528ad08732531041c0467724f788e18435a5f4af/dmscripts/generate_framework_agreement_signature_pages.py#L108-L110